styles/App.scss

```diff
@@ -30,3 +33,4 @@
 @import 'pages/events';
 @import 'pages/sponsors';
 @import 'pages/techgala';
+@import 'pages/internship';
```

Comment:
Same comment on modules; if the card CSS is reusable, might be good to move that into a component file.

Reply:
I can module the entire page, but I'll have to mess with `index.scss` and `global.scss`, since i use `content-section` and it's in index - whereas it should be in globals
If that won't interfere with any other PRs working on this, I'll go ahead and do it!
Another option is to for now duplicate the `content-section` code I need into `Internship.module.scss` and then module it - but that feels like unnecessary extra steps.

Reply:
You can use global code without it being moduled.

For example,

```js
<div className={`content-section ${styles['some-module-css-class']}`}>
...
```